### PR TITLE
Created views for users and posts

### PR DESCRIPTION
**Pull request by gabrielyea**
# Hello reviewer!
- Created views for all routes.
- Add minimal styling to views following wireframe.
- Created methods in models to access data.

## Database is seeded just run `rails db:setup` to generate all data!

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1 +1,13 @@
...
+  <ul class="postsList">
+   <%= render 'shared/user_card', user: @user %>
+      <% @posts.each do |p| %>
+        <%= link_to (user_post_path(p.author, p)) do %>
+          <li>
+            <%= render 'shared/post_card', post: p, max: 5 %>
+          </li>
+        <% end %>
+      <%end%>
+  </ul>
```
Comment (oxenprogrammer) [inline app/views/posts/index.html.erb]: **Changes Required**
- [x] Good effort with using `max` to return a limit of 5 comments. We should consider using a method from the models that returns the `most recent 5 comments` i.e something like `@post.recent_comments`

**Review by oxenprogrammer — CHANGES_REQUESTED** [review]
Hi @gabrielyea, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/index.html.erb (hunk 1 above)
Comment (gabrielyea) [reply app/views/posts/index.html.erb]: Done! added reference to `take_five_recent` method in partial  `_post_card`!

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉🎉

## Hi @gabrielyea, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**